server/models/development/update/synt.sql

```diff
@@ -115,3 +115,12 @@ UPDATE  `bhima`.`transaction_type` SET  `service_txt` =  'import_automatique' WH
 UPDATE  `bhima`.`transaction_type` SET  `service_txt` =  'group_deb_invoice' WHERE  `transaction_type`.`id` =5;
 UPDATE  `bhima`.`transaction_type` SET  `service_txt` =  'stock_loss' WHERE  `transaction_type`.`id` =13;
 UPDATE  `bhima`.`transaction_type` SET  `service_txt` =  'reversing_stock' WHERE  `transaction_type`.`id` =28;
+
+-- Add a test user (id:39) to user table
```

Comment:
I suggest to create a separate file like development/test/test.sql to put these instructions because, this synt file will be executed to modified the database stored in the production server.